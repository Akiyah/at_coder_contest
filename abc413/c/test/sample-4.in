20
1 1 10
1 1 20
1 1 30
1 1 40
1 1 50
2 1
2 1
2 1
2 1
2 1
1 2 10
2 2
1 2 20
2 2
1 2 30
2 2
1 2 40
2 2
1 2 50
2 2
