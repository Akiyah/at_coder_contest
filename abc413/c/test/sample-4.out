10
20
30
40
50
20
40
60
80
100
